kern/vmx.c

```diff
@@ -803,10 +803,9 @@ static u64 construct_eptp(unsigned long root_hpa)
 	u64 eptp;
 
 	/* TODO write the value reading from MSR */
-	eptp = VMX_EPT_DEFAULT_MT |
-		VMX_EPT_DEFAULT_GAW << VMX_EPT_GAW_EPTP_SHIFT;
+	eptp = VMX_EPTP_MT_WB | VMX_EPTP_PWL_4;
```

Comment:
Previous symbols were regiggered so we don't do manual bit shifting:

https://github.com/torvalds/linux/commit/bb97a016937816768ffb8cd7a49d9507e8a1bbda#diff-0a728b4244d136bb65f49f77858e0affR4301